### DISCUSSION TITLE
So hard for Chinese students to get students pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20439] [GitHub Education]
I'm a student from Shanghai Jiao Tong University, applying for Github Students Pack.
However, though I've tried my ID card, my school transcript, verification report applied by Ministry of Education(both in Chinese and English), none of them were accepted.
The system of Github can only recognize information in English, but it is quite hard for Chinese students to get a available paper in English. For example, the verification report by MOE is charged for 30 Yuan for English version (free for Chinese version) , and a transcript is charged for 5 Yuan.  What's more, after we pay for those papers, they are still rejected by Github.
When searching on the Internet, I've seen many Chinese students rejected for a couple of times, and there isn't any alternative method to submit those materials.
Please, a manual way is needed for us. The AI system is not fit for Chinese students.

**Comment by EddieMusick** [comment]
The system is awful and doesn't even work well for documents in English. I have pretty much given up on getting the student pack.

**Reply by EA-LGX** [reply]
我在2021年使用的是学生证和个人的邮箱号，一次就认证通过了。但是第二年申请的时候就变得非常困难了，什么学生证、学信网截图啥的都不管用，已经被拒绝了6次
